common/thanos/templates/_thanos_helpers.tpl

```diff
@@ -42,53 +42,61 @@ thanos-{{- $name -}}
 {{- $firstHost := first $root.Values.ingress.hostsFQDN -}}
 {{- required ".Values.ingress.hostsFQDN must have at least one hostname set" $firstHost -}}
 {{- else -}}
-thanos-{{- $name -}}.{{- required "$root.Values.global.region missing" $root.Values.global.region -}}.{{- required "$root.Values.global.tld missing" $root.Values.global.tld -}}
+thanos-{{ $name }}.{{- required "$root.Values.global.region missing" $root.Values.global.region -}}.{{- required "$root.Values.global.tld missing" $root.Values.global.tld -}}
 {{- end -}}
 {{- end -}}
 
 {{/* External gRPC URL of this Thanos. */}}
-{{- define "thanos.externalGrpcURL" -}}
+{{- define "thanos.grpcURL" -}}
```

Comment:
Why the renaming, pointing out the "externality" as we do with the "internality" makes it a better read for others I'd suppose.

Reply:
Yes - thought about the same. But I decided to just call it that, as `externalInternalURL` sound really weird to me. So just `internalURL` and `grpcURL`.

Reply:
technically an overall `s/external//g`

the `externalInternalURL` was a little bit funny indeed :D

Ok, I guess we can go with it.